css/style.css

```diff
@@ -465,3 +557,163 @@ footer {
   transition-duration: 0.7s;
   transform: scale(1.05);
 }
+
+.my_email {
+  font-size: 20px;
+  line-height: 28px;
+  text-align: center;
+  color: #ebebff;
+}
+
+@media screen and (min-width: 992px) and (max-width: 1100px) {
```

Comment:
- [ ] Your website is still on the mobile version. Kindly remove "` and (max-width: 11000px)`" so your desktop version design will be implemented.
- Your website:
<img width="698" alt="image" src="https://user-images.githubusercontent.com/77190203/150659875-0696a1cb-82ec-44b6-b613-41812a02d4f9.png">

Reply:
@medaminedev66 sorry but I'm confused, the last reviewer told me to implement it, and it works properly on my computer in the gap btween the sizes

Reply:
The previous reviewer said that the `min-width` should be between 992px and 1100px. He didn't mention anything about `max-width`.